docs/pywbemcligeneraloptions.rst

```diff
@@ -72,7 +70,7 @@ information on each command-group arguments and options):
 * **--output-format/-o** - Output format choice (Default: mof).
   Note that the actual output format may differ from this value because some
```

Comment:
What does "differ" mean? Is an output format that is not valid for a subcommand silently changed to something else? Should it not rather be rejected in that case?

Reply:
I think it is actually better for the user if we give it to them in the formats we allow instead of refusing this the request.  Thus, today class output does not have a table format. Should we make them change the output format if while they are in interactive mode to display a class?

Seems more logical to just do the display.  They will figure it out. 

**DISCUSSION**

Reply:
Also, if this is a real issue, Lets put it in its own issue since it will require code to change.

Reply:
Agfter discussion, it seems there is only one case where the output format is adjusted: when classes are shown and a table format is set. Given that we even intend to support table format for classes in 0.6.0, I suggest that we describe this one case as an exception.